Cargo.toml

```diff
@@ -2118,6 +2118,17 @@ description = "Pipe the output of one system into a second, allowing you to hand
 category = "ECS (Entity Component System)"
 wasm = false
 
+[[example]]
+name = "state_scoped"
+path = "examples/ecs/state_scoped.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.state_scoped]
+name = "State Scoped"
+description = "Show how to use state scoped systems"
```

Comment:
This needs a better description. It should explain what the feature is to users who don't know the jargon :)